docs/configuration/protocols/ospf.rst

```diff
@@ -19,20 +19,90 @@ OSPF is a widely used IGP in large enterprise networks.
 OSPFv2 (IPv4)
 #############
 
-In order to have a VyOS system exchanging routes with OSPF neighbors, you will
-at least need to configure an OSPF area and some network.
+.. cfgcmd:: set protocols ospf area <number>
 
-.. code-block:: none
+   This command is udes to enable the OSPF process. The area number can be 
+   specified in decimal notation in the range from 0 to 4294967295. Or it
+   can be specified in dotted decimal notation similar to ip address.
 
-  set protocols ospf area 0 network 192.168.0.0/24
+.. cfgcmd:: set protocols ospf area <number> network A.B.C.D/M
```

Comment:
please add `< >` to all variables in a command. Its important for the coverage page.
`.. cfgcmd:: set protocols ospf area <number> network <A.B.C.D/M>`

Reply:
Done. Actually you could do it yourself :-)